### DISCUSSION TITLE
Cannot access remote API server for credit data

### DISCUSSION THREAD

**Discussion by abhikmr778** [DevSkillsHQ/discusssions#34] [Help] [answered]
Hi all

I am working on Lookup Service - Level 2 which tasks me to build a backend service that implements the [Lookup Service REST API](https://infra.devskills.app/lookup/api/1.0.0) and integrates with the [Credit Data REST API](https://infra.devskills.app/credit-data/api/1.0.0) to aggregate historical credit data. I am new to performing such tasks.

I am not sure where to get the credit data as I am not able to access the remote API https://infra.devskills.app/api/credit-data.
Am I trying to access the correct API? From where can I get the credit data?

Any help is appreciated. 
Best Regards
Abhinav

**Comment by fenske** [comment, marked as answer]
Hi @abhikmr778. Yeap, the endpoint is correct.

So if you check the spec, it is the following for the server:
```
"servers": [
    {
      "url": "https://infra.devskills.app/api/credit-data"
    }
  ]
```

And then, check what individual endpoints it has, e.g.:

```
"/debt/{ssn}"
```

Now check the definition of the API tests, you'll see they go against a number of ssn values, e.g. 287-54-7823

Now try calling https://infra.devskills.app/api/credit-data/debt/287-54-7823.

Hope this help.

I wish you the best of luck with the rest of your interview process!

**Reply by abhikmr778** [reply]
Oh, that helps a lot. Thank you very much :)
